### PR TITLE
fix(DMVP-5908): fix rules 

### PR DESCRIPTION
**Pull request by sophie-dasmeta-com**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -65,5 +60,22 @@ variable "plan_selection_tag" {
...
\ No newline at end of file
+variable "rules" {
+  description = "List of rules to attach to the plan"
+  type        = list(map)
+  default = [
+    {
+      name              = "daily"
+      schedule          = "cron(0 12 * * ? *)"
+      continuous_backup = true
+      vault             = "Backup"
```
Comment (mrdntgrn) [inline variables.tf:67]: seems you do not use `continuous_backup` and `vault` params,
as I understood it is supposed they will replace var.enable_continuous_backup and local.vault_name ones ?

same for recovery_point_tags param one, and in code you use
```terraform
recovery_point_tags = {
      Environment = var.env
    }
```

Comment (mrdntgrn) [reply variables.tf:67]: the aws_backup_plan.name its  rule name and aws_backup_selection resource block names and name params all have `daily` hardcoded naming, can we maybe have this namings more general/configurable?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# terraform-aws-backup
```
Comment (mrdntgrn) [inline README.md:1]: Please have some basic description what this module does and some simple example code here, have also tests/basic folder with some basic setups example

File: monitoring.tf [hunk 3]
```diff
@@ -135,13 +129,13 @@ locals {
...
-  count             = var.enable_sns_notifications ? 1 : 0
-  backup_vault_name = local.vault_name
-  sns_topic_arn     = module.sns_topic.sns_topic_arn
-  backup_vault_events = [
-    "BACKUP_JOB_COMPLETED", # filter successful backups on sns subscription!
-    "RESTORE_JOB_STARTED", "RESTORE_JOB_COMPLETED",
-    "S3_BACKUP_OBJECT_FAILED", "S3_RESTORE_OBJECT_FAILED"
-  ]
-}
+# resource "aws_backup_vault_notifications" "this" {
```
Comment (mrdntgrn) [inline monitoring.tf]: why we have this block commented?
if there is no need of it please remove,
if you need it to keep commented have a comment/description why

Comment (mrdntgrn) [reply monitoring.tf]: also please checn and remove the variables which we do not use like
enable_continuous_backup
backup_schedule

Comment (sophie-dasmeta-com) [reply monitoring.tf]: fixed these

**Review by mrdntgrn — APPROVED** [review]

**Pull request merged**